core/src/main/java/org/apache/james/core/quota/QuotaModelModule.java

```diff
@@ -0,0 +1,55 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.core.quota;
+
+import org.apache.james.core.quota.quotacomponent.JmapUploadQuotaComponent;
+import org.apache.james.core.quota.quotacomponent.MailBoxQuotaComponent;
+import org.apache.james.core.quota.quotacomponent.SieveQuotaComponent;
+import org.apache.james.core.quota.quotascope.DomainQuotaScope;
+import org.apache.james.core.quota.quotascope.GlobalQuotaScope;
+import org.apache.james.core.quota.quotascope.UserQuotaScope;
+import org.apache.james.core.quota.quotatype.CountQuotaType;
+import org.apache.james.core.quota.quotatype.SizeQuotaType;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Scopes;
+import com.google.inject.multibindings.Multibinder;
+
+public class QuotaModelModule extends AbstractModule {
```

Comment:
This has NOTHING to do in `/core`. If needed move this to `/server/container/guice`